src/graphql/operations/proposals.ts

```diff
@@ -65,6 +65,16 @@ export default async function (parent, args) {
     params.push(verifiedSpaces);
   }
 
+  if (where.flagged === true) {
+    searchSql += ' AND p.id in (?)';
```

Comment:
Sql arguments should be in uppercase
```suggestion
    searchSql += ' AND p.id IN (?)';
```